app/views/content/train-to-be-a-teacher/if-your-application-is-unsuccessful.md

```diff
@@ -0,0 +1,79 @@
+---
+title: If your teacher training application is unsuccessful
+subcategory: Your teacher training application
+description: |-
+  Read about what to do next if your teacher training application is unsuccessful. Find advice on the different reasons an application can be unsuccessful and how to get support.
+related_content:
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
+    7 things to include in your teacher training application : "/blog/7-things-to-include-in-your-teacher-training-application"
+    11 top tips for interview success : "/blog/11-top-tips-for-interview-success"
+date: "2023-03-30"
+keywords:
+  - adviser
+  - advisor
+  - ITT
+  - Initial Teacher Training
+  - QTS
+  - Qualified Teacher Status
+  - Teacher Training Adviser
+  - application
+  - apply
+  - how to apply
+  - applying for teacher training
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+---
+
+Try not to worry - you can reapply for up to 4 courses at a time, as many times as you want. Many people are successful the second or third time around.
+
+It’s worth looking at the feedback the course provider has given to understand why your application was not accepted. This will help you to decide on your next steps.
+
+There could be a number of reasons your application was unsuccessful:
+
+### The course is full
```

Comment:
I think these should be h2s rather than h3s, if there isn't a h2 nested above them